islandora_rdm_disk_usage/src/Controller/DiskUsageController.php

```diff
@@ -11,226 +11,242 @@
+            $count[$owner] = $count[$owner] + $size;
+          }
+          else {
+            $count[$owner] = $size;
+          }
         }
-        /* $report['content']['pageTitle'] = [
-          '#type' => 'markup',
-          '#markup' => '<h3> Group by ' . ucfirst($groupBy) . '</h3>',
-          ]; */
-        $report = [
-            '#data' => [
-                'labels' => array_keys($count),
-                'datasets' => [
-                    [
-                        'label' => 'Media Items',
-                        'data' => array_values($count),
-                        'backgroundColor' => ['#00557f', '#f8413c', '#666666'],
-                        'hoverBackgroundColor' => ['#004060', '#9b2926', '#333333'],
-                    ],
-                ],
-            ],
-            '#graph_type' => $chartType,
-            '#id' => 'my' . $chartType . $groupBy,
-            '#type' => 'chartjs_api',
-        ];
-        return $report;
-    }
-
-    /**
-     * Query with the entity_type.manager, load entities in Drupal code.
-     */
-    private function getMyList() {
-        $query = $this->entityTypeManager->getStorage('media');
-
-        // Get An array of entity ID of all published materials.
-        $query_all = $query->getQuery()
-                ->condition('status', 1)
-                ->sort('field_file_size', 'DESC')
-                ->execute();
-
-        // Load the entities, loadMultiple($ids):
-        // array $ids: An array of entity ID, or NULL to load all entities.
-        $list = \Drupal::entityTypeManager()
-                ->getStorage('media')
-                ->loadMultiple($query_all);
-        // ksm($list); //view the value on the page.
-        return $list;
+      }
+      else {
+        continue;
+      }
     }
-
-    // Helper method to decide if a type exists in the array.
-
-    /**
-     * Var: $type: string, $array: array; return Boolean.
-     */
-    private function exist($type, $array) {
-        foreach ($array as $key => $value) {
-            if ($key == $type) {
-                return TRUE;
-            }
-        }
-        return FALSE;
+    /* $report['content']['pageTitle'] = [
+    '#type' => 'markup',
+    '#markup' => '<h3> Group by ' . ucfirst($groupBy) . '</h3>',
+    ]; */
+    $report = [
+      '#data' => [
+        'labels' => array_keys($count),
+        'datasets' => [
+                  [
+                    'label' => 'Media Items',
+                    'data' => array_values($count),
+                    'backgroundColor' => ['#00557f', '#f8413c', '#666666'],
+                    'hoverBackgroundColor' => ['#004060', '#9b2926', '#333333'],
+                  ],
+        ],
+      ],
+      '#graph_type' => $chartType,
+      '#id' => 'my' . $chartType . $groupBy,
+      '#type' => 'chartjs_api',
+    ];
+    return $report;
+  }
+
+  /**
+   * Query with the entity_type.manager, load entities in Drupal code.
+   */
+  private function getMyList() {
```

Comment:
Change this function name to something more descriptive. 'My[thing]' should only ever be used in example code.